zeeguu_api/api/user_languages.py

```diff
@@ -0,0 +1,155 @@
+import flask
+import zeeguu
+from flask import request
+from zeeguu.model.language import Language
+from zeeguu.model.user_language import UserLanguage
+
+
+from .utils.route_wrappers import cross_domain, with_session
+from .utils.json_result import json_result
+from . import api
+
+session = zeeguu.db.session
+
+USER_LANGUAGES = "user_languages"
+MODIFY_USER_LANGUAGE = "user_languages/modify"
+DELETE_USER_LANGUAGE = "user_languages/delete"
+INTERESTING_LANGUAGES = "user_languages/interesting"
+READING_LANGUAGES = "user_languages/reading"
+
+
+# ---------------------------------------------------------------------------
+@api.route(f"/{MODIFY_USER_LANGUAGE}", methods=("POST",))
+# ---------------------------------------------------------------------------
+@cross_domain
+@with_session
+def add_user_language():
```

Comment:
would be nice to have the method named as the endpoint.

Reply:
e.g. `modify_user_language`

Reply:
Agreed, done